src/__tests__/index.node.js

```diff
@@ -58,17 +59,26 @@ test('Server logger', async t => {
 
 test('Server logger listening on events', async t => {
   let called = false;
-  class Transport {
+  class Transport extends TransportStream {
     name: string;
 
     constructor() {
+      super();
       this.name = 'test-transport';
     }
-    log(level: SupportedLevelsType, message: string, meta: mixed) {
+    log({
+      level,
+      message,
+      hello,
+    }: {
+      level: SupportedLevelsType,
+      message: string,
+      hello: string,
+    }) {
       t.equals(level, 'info', 'level is ok');
       t.equals(message, 'test', 'message is ok');
       t.equals(message, 'test', 'message is ok');
-      t.deepLooseEqual(meta, {hello: 'world'}, 'meta is ok');
+      t.equals(hello, 'world', 'meta is ok');
```

Comment:
winston v3 no longer maintains a separate meta field, and instead merges everything.